apps/docs/src/components/Downloads/SlideChart.tsx

```
@@ -0,0 +1,424 @@
+    }
+
+    return `M ${startOuterX} ${startOuterY} A ${outerR} ${outerR} 0 ${largeArc} 1 ${endOuterX} ${endOuterY} L ${startInnerX} ${startInnerY} A ${innerR} ${innerR} 0 ${largeArc} 0 ${endInnerX} ${endInnerY} Z`;
+  };
+
+  return (
+    <svg width={width} height={height} className="overflow-visible">
+      {data.map((point, index) => {
+        const angle = (point.value / total) * 2 * Math.PI;
+        const startAngle = currentAngle;
+        const endAngle = currentAngle + angle;
+        currentAngle = endAngle;
```

ESLint

[error] Error: Cannot reassign variable after render completes

Reassigning `currentAngle` after render has completed can cause inconsistent behavior on subsequent renders. Consider using state instead.

/home/jailuser/git/apps/docs/src/components/Downloads/SlideChart.tsx:266:9
  264 |         const startAngle = currentAngle;
  265 |         const endAngle = currentAngle + angle;
> 266 |         currentAngle = endAngle;
      |         ^^^^^^^^^^^^ Cannot reassign `currentAngle` after render completes
  267 |
  268 |         const color = getColor(index, point, colors);
  269 |

(react-hooks/immutability)